zingolib/src/utils.rs

```diff
@@ -31,10 +32,11 @@ macro_rules! build_push_list {
     };
 }
 
+#[cfg(any(test, feature = "test-elevation"))]
```

Comment:
At some point can we combine these in way that permits a single `cfg` attribute to gate all of them?